remote_test.go

```diff
@@ -1210,6 +1212,41 @@ func (s *RemoteSuite) TestList(c *C) {
 	}
 }
 
+func (s *RemoteSuite) TestListPeeling(c *C) {
+	remote := NewRemote(memory.NewStorage(), &config.RemoteConfig{
+		Name: DefaultRemoteName,
+		URLs: []string{"https://github.com/aide/aide.git"},
```

Comment:
```suggestion
		URLs: []string{"https://github.com/git-fixtures/tags.git"},
```

Running `go test -v . -check.v -check.f=RemoteSuite.TestListPeeling` with https://github.com/git-fixtures/tags.git instead still passes.